versions.tf

```diff
@@ -7,7 +7,7 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = ">=3.85.0"
+      version = ">= 3.112.0"
```

Comment:
Any reason why we have used `>=3.112.0` here?

Reply:
Updated